cmd/run.go

```diff
@@ -78,6 +78,18 @@ type DatastoreConfig struct {
 	// old keys. The storage cache is used to cache query results for various static resources
 	// such as type definitions.
 	MaxCacheSize int
+
+	// MaxOpenConns is the maximum number of open connections to the database.
+	MaxOpenConns int
+
+	// MaxIdleConns is the maximum number of connections in the idle connection pool.
+	MaxIdleConns int
+
+	// ConnMaxIdleTime is the maximum amount of time a connection may be idle.
+	ConnMaxIdleTime time.Duration
+
+	// ConnMaxLifetime is the maximum amount of time a connection may be reused.
```

Comment:
```suggestion
	// ConnMaxLifetime is the maximum amount of time a connection to the datastore may be reused.
```

Reply:
(same suggestion for the .config-schema.json file)

Reply:
Thank you! Fixed.